_harp/public/pages/css.md

```diff
@@ -94,37 +50,15 @@ You'll notice the example above follows these guidelines:
 6. The closing brace is at the same indent level as the selector(s)
 7. Double quotes are used for strings
 
-<!---
-Additionally, properties should ideally be ordered as follows (some properties omitted for brevity):
-
-1. display
-2. position
-3. [dimensions]
-	1. width
-	2. max-width
-	3. height
-	4. max-height
-4. [box]
-	1. margin
-	2. padding
-	3. box-sizing
-5. [positioning]
-	1. top
-	2. left
-	3. right
-	3. bottom
-6.
--->
-
 ### Basic Selectors
 
 There are six types of selectors: **element selectors**, **IDs***, **classes**, **attribute selectors**, **pseudo-class** and **pseudo-elements**.
 
 #### ID Selectors
 
-Selectors using IDs are not permitted in Cuberis stylesheets in order to avoid specificity issues and ensure the maintanability and extensibility of codebases over time by multiple developers. Existing classes should be used if possible, or a class should be added to the element if no suitable one exists.
+ID selectors are discouraged in order to avoid specificity issues. Existing classes should be used if possible, or a class should be added to the element if no suitable one exists.
 
-IDs are reserved for targeting DOM elements with javascript. View the [javascript standards page](#) for naming conventions and more information.
+IDs should generally be reserved for targeting DOM elements via JavaScript. View the [JavaScript page](#) for naming conventions and more information.
```

Comment:
They are also used in relationship / roles (in conjunction with JS)... thinking Bootstrap components for example.

Reply:
I'll prob leave it b/c I think all that falls under the category of "targeting"